scripts/auto_translate.py

```
@@ -0,0 +1,1116 @@
+import os
+import json
+import requests
+import base64
+import difflib
+import re
```

Ruff (0.8.2)

`re` imported but unused

Remove unused import: `re`

(F401)